development/contracts/WhitelistArbitrator.sol

```diff
@@ -123,7 +122,7 @@ contract WhitelistArbitrator is BalanceHolder {
     function requestArbitration(bytes32 question_id, uint256 max_previous)
     external payable returns (bool) {
 
-        uint256 arbitration_fee = getDisputeFee(question_id);
+        uint256 arbitration_fee = getDisputeFee();
```

Comment:
Leave the question_id in there, it's designed to give the arbitrator flexibility even if it's not always used